docs/docs/reference/metaprogramming/macros.md

```diff
@@ -617,7 +617,7 @@ It is possible to deconstruct or extract values out of `Expr` using pattern matc
 `scala.quoted` contains objects that can help extracting values from `Expr`.
 
 * `scala.quoted.Const`/`scala.quoted.Consts`: matches an expression of a literal value (or list of values) and returns the value (or list of values).
-* `scala.quoted.Value`/`scala.quoted.Values`: matches an expression of a value (or list of values) and returns the value (or list of values).
+* `scala.quoted.Expr`/`scala.quoted.Exprs`: matches an expression of a value (or list of values) and returns the value (or list of values).
```

Comment:
This is not intuitive --- `Expr`  and values are very different, conceptually.

Reply:
If we say that `Expr(x)` takes a value and lifts it into and `Expr`, then `case Expr(x) => ` takes the value out of the expression. It seems really intuitive but may need to be explained in a better way.

Reply:
Maybe `unlift`, to align with `Unliftable`?

Reply:
Then we would need to move `Expr.apply` to `Lift.apply` (or `Lifted.apply`). This would break a lot of code.

In a patterns `Unlifted` works better, which is a bit too long.

Reply:
I will open a PR with the alternative

Reply:
See #8513. It does look semantically clearer. But has migration costs (a deprecation warining).